py/picca/io.py

```diff
@@ -1404,15 +1404,15 @@ def read_blinding(in_dir):
                                                            + '/*.fits.gz')
     filename = files[0]
     hdul = fitsio.FITS(filename)
-    header = hdul[1].read_header()
-    # new runs of picca_deltas should have a blinding keyword
-    if "BLINDING" in header:
-        blinding = header["BLINDING"]
-    # older runs are not from DESI main survey and should not be blinded
-    else:
-        blinding = "none"
-
-    return blinding
+    if 'LAMBDA' not in hdul: # This is for BinTable format
+        header = hdul[1].read_header()
+        # older runs not from DESI main survey and should not be blinded
+        if "BLINDING" not in header: 
```

Comment:
They way it is written actually saves an else. Or loading two headers in case is image model. But of course I don't mind leaving the 'if' instead of 'if not.'